### DISCUSSION TITLE
vant4 uniapp

### DISCUSSION THREAD

**Discussion by yu5056561661** [youzan/vant#12165] [General]
vant4 uniapp 可以使用吗？ 我的的 uniapp v3+ts+vite 导入后进去都是白屏

**Comment by wjw-gavin** [comment]
请参考：[是否支持在 uni-app 中使用](https://vant-contrib.gitee.io/vant/#/zh-CN/faq#shi-fou-zhi-chi-zai-uni-app-zhong-shi-yong)

**Comment by ghost** [comment]
所以可以用吗？
